test/unit/whitehall/publishing_api_test.rb

```diff
@@ -22,6 +28,42 @@ class Whitehall::PublishingApiTest < ActiveSupport::TestCase
     assert_requested request
   end
 
+  test "#publish sends unpublishing for case studies to the content store" do
+    edition = create(:draft_case_study)
+    unpublishing = create(:unpublishing, edition: edition)
+
+    payload = PublishingApiPresenters::Unpublishing.new(unpublishing).as_json
+    request = stub_publishing_api_put_item(unpublishing.document_path, payload)
+
+    Whitehall::PublishingApi.publish_async(unpublishing)
+
+    assert_requested request
+  end
+
+  test "#publish skips sending unpublishings for formats other than case study" do
+    edition = create(:draft_publication)
+    unpublishing = create(:unpublishing, edition: edition)
+
+    Whitehall::PublishingApi.publish_async(unpublishing)
+  end
+
+  test "#publish sends case studies to the content store" do
+    edition = create(:published_case_study)
+
+    presenter = PublishingApiPresenters.presenter_for(edition)
+    request = stub_publishing_api_put_item(presenter.base_path, presenter.as_json)
+
+    Whitehall::PublishingApi.publish_async(edition)
+
+    assert_requested request
+  end
+
+  test "#publish skips sending policies to content store" do
+    policy = create(:published_policy)
+
+    Whitehall::PublishingApi.publish_async(policy)
```

Comment:
nitpick: i would have preferred an `assert_not_requested` here for the test to look complete. doesn't block from merging this PR.